### DISCUSSION TITLE
jQuery with Webpack jQuery is undefined at first time

### DISCUSSION THREAD

**Discussion by SamShekhovtsov** [codesandbox/codesandbox-client#4547] [General]
it might be that I miss something, but jQuery support within projects that use Webpack is nice to have feature. However, currently if I try to use jQuery with Webpack - it shows "ReferenceError": "jQuery is not defined". If all code lines where jQuery is referenced are commented out and applications compiles fine, than the code can be uncommented and everything works fine. The problem only happens before the first compilation, which is weird. For example, following sandbox: https://codesandbox.io/s/jolly-aryabhata-fx4ue

is there something I should add or change in order to make jQuery available before the first compilation?

**Comment by CodeSandboxTeam** [comment]
Hello Sem,


If we're not mistaken you're trying to use webpack plugins, where we use our own bundler dubbed sandpack. It therefore wouldn't work. Don't think our bundler supports plugins. What are you trying to build?


–Oskar



--- original message ---
On July 13, 2020, 7:07 PM GMT+2 notifications@github.com wrote:



it might be that I miss something, but jQuery support within projects that use Webpack is nice to have feature. However, currently if I try to use jQuery with Webpack - it shows "ReferenceError": "jQuery is not defined". If all code lines where jQuery is referenced are commented out and applications compiles fine, than the code can be uncommented and everything works fine. The problem only happens before the first compilation, which is weird. For example, following sandbox: https://codesandbox.io/s/jolly-aryabhata-fx4ue

is there something I should add or change in order to make jQuery available before the first compilation?

—

You are receiving this because you are subscribed to this thread.

Reply to this email directly, view it on GitHub, or unsubscribe.
--- end of original message ---
